metricbeat/module/ceph/mgr_osd_disk/mgr_osd_disk.go

```diff
@@ -0,0 +1,93 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package mgr_osd_disk
+
+import (
+	"fmt"
+
+	"github.com/elastic/beats/metricbeat/helper"
+	"github.com/elastic/beats/metricbeat/mb"
+	"github.com/elastic/beats/metricbeat/mb/parse"
+)
+
+const (
+	defaultScheme      = "https"
+	defaultPath        = "/request"
+	defaultQueryParams = "wait=1"
+
+	cephPrefix = "df"
+)
+
+var (
+	hostParser = parse.URLHostParserBuilder{
+		DefaultScheme: defaultScheme,
+		DefaultPath:   defaultPath,
+		QueryParams:   defaultQueryParams,
+	}.Build()
+)
+
+func init() {
+	mb.Registry.MustAddMetricSet("ceph", "mgr_osd_disk", New,
```

Comment:
Actually I'm facing issues naming this one with an old name. Not sure if it's doable:

```
panic: metricset 'ceph/cluster_disk' is already registered
```

Reply:
Oh, I meant to change it from `mgr_osd_disk` to `mgr_pool_disk`, in case it is exposing the same kind of data as the `pool_disk` metricset. If it is not the same data I am ok with keeping it as `mgr_osd_disk`. And in any case I would leave the `mgr_` prefix or any other prefix that distinguishes it from the old metricsets.

Reply:
Fixed.